src/main/resources/publicationFrame.json

```diff
@@ -46,6 +46,7 @@
     "isOpenAccess": "openAccess",
     "isPeerReviewed": "peerReviewed",
     "hasDoi": "doi",
+    "hasDoiRequest": "doiRequest",
```

Comment:
I was not sure about adding this line, but it seemed appropriate.

Reply:
Yes, it is for the mapping to the external model